docs/protocol-design/introduction.md

```diff
@@ -0,0 +1,66 @@
+Title: Protocol Design Introduction | Nano Documentation
+Description: As the beginning of the living whitepaper, get details and background of the problems Nano aims to solve that other digital money hasn't been able to
+
+# Protocol Design - Introduction
+
+--8<-- "wip-living-whitepaper.md"
+
+--8<-- "contributing-code.md"
```

Comment:
Should we move this to the node implementation introduction (or a link to the Contributing page)? In this page it would make sense to have a callout for contributing to the protocol instead.

Reply:
Updated